tool/template-learn-uses.html

```diff
@@ -72,22 +72,22 @@ <h2 class="mb-10">{% trans %}Digital Cryptocurrency Wallets{% endtrans %}</h2>
 
     <section class="row mb-50">
       <h6 class="section-marker">{% trans %}Exchanges{% endtrans %}</h6>
-      <div class="col-md-4 offset-md-1 mb-10 text-center">
+      <div class="col-md-4 offset-md-1 mb-10 text-center text-md-left">
         <img class="mw-100" src="./img/uses/exchanges.png">
       </div>
-      <div class="col-md-4 offset-md-1">
+      <div class="col-md-4 offset-md-2">
         <h2 class="mb-10">{% trans %}Exchanges{% endtrans %}</h2>
         <p class="mb-10 normal">{% trans %}Market makers and individuals use exchanges to trade XRP and other digital assets. Exchanges are often also intermediaries in cross-border payments among financial institutions.{% endtrans %}</p>
         <a href="exchanges.html" class="btn btn-outline-secondary">{% trans %}Explore Exchanges{% endtrans %}</a>
       </div>
     </section>
 
-    <section class="row last-section">
+    <section class="row mb-50">
```

Comment:
This undoes some changes I put in specifically to make the spacing at the bottom of the page match Gratas' request. Is that intentional?

Reply:
Did not know that. We can leave as is, but the spacing is more or less the same with one less class